src/Tribe/Admin/Troubleshooting.php

```diff
@@ -0,0 +1,216 @@
+                'do_menu_page',
+            ]
+        );
+    }
+
+    /**
+     * Adds a link to the the WP admin bar
+     */
+    public function add_toolbar_item()
+    {
+        $capability = apply_filters('tribe_events_troubleshooting_capability', 'install_plugins');
+
+        // prevent users who cannot install plugins from seeing addons link
+        if (current_user_can($capability)) {
+            global $wp_admin_bar;
+
+            $wp_admin_bar->add_menu([
+                'id'     => 'tribe-events-troubleshooting',
+                'title'  => esc_html__('Event Add-Ons', 'tribe-common'),
+                'href'   => Tribe__Settings::instance()->get_url([ 'page' => self::MENU_SLUG ]),
+                'parent' => 'tribe-events-settings-group',
+            ]);
+        }
+    }
+
+    /**
+     * Checks if the current page is the troubleshooting page
+     *
+     * @since TBD
+     * 
+     * @var string
+     *
+     * @return bool
+     */
+    public function is_current_page()
+    {
+        if (! Tribe__Settings::instance()->should_setup_pages() || ! did_action('admin_menu')) {
+            return false;
+        }
+
+        if (is_null($this->admin_page)) {
+            _doing_it_wrong(
+                __FUNCTION__,
+                'Function was called before it is possible to accurately determine what the current page is.',
+                '4.5.6'
+            );
+            return false;
+        }
+
+        return Tribe__Admin__Helpers::instance()->is_screen($this->admin_page);
+    }
+
+    /**
+     * Renders the Troubleshooting page
+     * 
+     * @since TBD
+     * 
+     * @var string
+     */
+    public function do_menu_page() {
+        $main = Tribe__Main::instance();
+        include_once Tribe__Main::instance()->plugin_path . 'src/admin-views/troubleshooting.php';
+    }
+
+    public function is_any_issue_active() {
+        $issues = $this->get_issues_found();
+        $active_issues = wp_list_pluck( $issues, 'active' );
+        return in_array( true, $active_issues );
+    }
+
+    public function is_active_issue( $slug ) {
+        if ( 'timezone' === $slug ) {
+            return Timezones::is_utc_offset( Timezones::wp_timezone_string() );
+        }
+        if ( 'install-max' === $slug ) {
+            
+        }
+        return false;
+    }
+
+    public function get_issues_found() {
+        $issues_found = apply_filters( 'tec_help_troubleshooting_issues_found', [
+            [
+                'title' => __('Time zone is not set', 'tribe-common'),
+                'description' => __('We recommend that our users use a location time zone and avoid using UTC offsets.', 'tribe-common'),
+                'more_info' => 'https://evnt.is/somewhere',
+                'fix' => 'https://evnt.is/somewhere',
+                'active' => $this->is_active_issue( 'timezone' ),
+            ],
+            [
+                'title' => __('Install max has been reached', 'tribe-common'),
```

Comment:
don't forget to replace placeholder text

Reply:
I've updated this section with the text provided by Leah.